tests/qinfo/test_fisher.py

```diff
@@ -530,11 +528,13 @@ def circ(params):
     @pytest.mark.torch
     @pytest.mark.tf
     @pytest.mark.jax
-    @pytest.mark.xfail
     def test_consistency(self):
```

Comment:
This test is skip on the CI checks, you need to mark this one with `@pytest.mark.all_interfaces`

Reply:
@eddddddy You can change that and then merge 👍